PSScriptAnalyzerRules.psd1

```diff
@@ -0,0 +1,84 @@
+@{
+    Severity     = @('Error')
+    IncludeRules = @(
+        'PSUseCompatibleSyntax',
+        'PSAvoidUsingCmdletAliases',
+        'PSAvoidUsingWriteHost',
+        'PSAvoidDefaultValueSwitchParameter',
+        'PSReservedCmdletChar',
+        'PSReservedParams',
+        'PSAvoidUsingUserNameAndPassWordParams',
+        'PSAvoidUsingPlaintTextForPassword',
+        'PSAvoidUsingWMICmdlet',
+        'PSAvoidUsingWriteHost',
+        'PSMisleadingBacktick',
+        'PSMissingModuleManifestField',
+        'PSPossibleIncorrectComparisonWithNull',
+        'PSUseApprovedVerbs',
+        'PSUseOutputTypeCorrectly',
+        'PSShouldProcess',
+        'PSUserToExportFieldsInManifest',
+        'PSUseSingularNouns',
+        'PSAvoidUsingInvokeExpression',
+        'PSUseShouldProcessForStateChangingFunctions',
+        'PSUseDeclaredVarsMoreThanAssignments',
+        'PSUseCore',
+        'PSPlaceOpenBrace',
+        'PSPlaceCloseBrace',
+        'PSUseConsistentWhitespace',
+        'PSUseConsistentIndentation',
+        'PSAlignAssignmentStatement'
+        # 'PSUseCorrectCasing' ## We do not like the use of -EQ, prefer -eq
```

Comment:
```suggestion
        'PSAlignAssignmentStatement',
        'PSUseCorrectCasing'
```
I do not dislike `PSUseCorrectCasing`. I prefer `-eq` over `-EQ`

Reply:
I believe PSUseCorrectCasing forces -EQ. We prefer in -eq in dbatools and had to disable this rule because Invoke-DbatoolsFormatter, which uses the script analyzer, would force it to -EQ when this rule was enabled. That may be an old version though.

Reply:
Please feel free to commit your change and merge -- I was unable to find a clear answer but that is what we removed to address the -EQ issue.

Reply:
It seems that PSScriptAnalyzer does not say anything about the operators as far as I can try. No alerts are shown for `-eq`, `-EQ` or `-eQ`. Well, I guess it doesn't matter either way, but `settings.json` contains `"powershell.codeFormatting.useCorrectCasing": true`, so it might be better to match it.